### PR TITLE
newdevice: Added support for Teradici PCoIP card

### PR DESCRIPTION
**Pull request by EnzoZafra**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/teradici-pcoip.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+os: teradici-pcoip
+text: 'Teradici PCoIP'
+type: workstation
+icon: teradici
+mib_dir:
+    - teradici
+over:
+    - { graph: device_bits, text: 'Device Bits' }
+    #- { graph: device_pcoip-net-latency, text: 'Net Latency' }
```
Comment (laf) [inline includes/definitions/teradici-pcoip.yaml]: can you remove the two commented out lines please.

File: includes/polling/os/teradici-pcoip.inc.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+//SNMPv2-SMI::enterprises.25071.1.2.6.1.1.5.1 = STRING: "TERA2240 revision 1.0 (512 MB)"
+//SNMPv2-SMI::enterprises.25071.1.2.6.1.1.6.1 = STRING: "AHK TERA2240 4x DP Host Card Copper"
+//SNMPv2-SMI::enterprises.25071.1.2.6.1.1.7.1 = STRING: "DS12144FA"
+//SNMPv2-SMI::enterprises.25071.1.2.6.1.1.8.1 = STRING: "DXM13-9 PV2.0.D"
+//SNMPv2-SMI::enterprises.25071.1.2.6.1.1.9.1 = STRING: "4.7.5"
+
+$hardware = trim(snmp_get($device, '1.3.6.1.4.1.25071.1.2.6.1.1.5.1', '-OQv', '', ''), '" ');
```
Comment (laf) [inline includes/polling/os/teradici-pcoip.inc.php]: You don't need the `, '', ''` in the snmp_get call, those are optional.

File: includes/definitions/teradici-pcoip.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+os: teradici-pcoip
+text: 'Teradici PCoIP'
+type: workstation
```
Comment (laf) [inline includes/definitions/teradici-pcoip.yaml]: Sorry didn't spot this earlier. 

If you are adding a new type then this needs to go into includes/definitions.inc.php (see network, firewall, appliance as examples).

Comment (EnzoZafra) [inline includes/definitions/teradici-pcoip.yaml]: I see lines that point to an icon for device types, i.e. `$config['device_types'][$i]['icon'] = 'server.png';`
Where are these stored? I could not find them

Comment (laf) [inline includes/definitions/teradici-pcoip.yaml]: For now that line isn't used but it's best to keep it until we either add support or remove those config lines.

**Pull request merged**